lib/src/consumer.dart

```diff
@@ -156,7 +158,7 @@ class Consumer<T> extends SingleChildStatelessWidget {
   /// {@template provider.consumer.constructor}
   /// Consumes a [Provider<T>]
   /// {@endtemplate}
-  Consumer({
+  const Consumer({
```

Comment:
Consumer and Providers are purposefully not using `const` because of the builder syntax (const is impossible to use)